common/services/github.js

```diff
@@ -37,7 +37,7 @@ async function _getPullReviewsFromGithub(label) {
 
   try {
     const { data } = await octokit.search.issuesAndPullRequests({
-      q: `is:pr+is:open+archived:false+user:${owner}+label:${label}`,
+      q: `is:pr+is:open+archived:false+draft:false+user:${owner}+label:${label}`,
```

Comment:
Cette chaine de caractères commence à faite beaucoup de choses, je propose.

```suggestion
      const includeDrafts = false;
      const filter = `is:pr+is:open+archived:false+${includeDrafts}+user:${owner}+label:${label}`
      (..)
      const { data } = await octokit.search.issuesAndPullRequests({ 
      	q: filter, sort: 'updated', order: 'desc',
    });
```

Reply:
Je propose de faire une autre PR pour améliorer la construction de ce filtre

Reply:
See https://github.com/1024pix/pix-bot/pull/152